### DISCUSSION TITLE
The service "Divante\TrackingBundle\EventListener\TrackingListener" has a dependency on a non-existent service "pimcore.service.request.pimcore_context_resolver".

### DISCUSSION THREAD

**Discussion by akashhaldwani** [pimcore/pimcore#13486] [Q&A / Support]
After installing "divante-ltd/pimcore5-user-tracking" I had activated plugin via "Bundle and Brick Tab"
and than an error comming "The service "Divante\TrackingBundle\EventListener\TrackingListener" has a dependency on a non-existent service "pimcore.service.request.pimcore_context_resolver"."

Please guide me to resolve this issue.

![image](https://user-images.githubusercontent.com/96160558/198833694-d6f3f2aa-89fa-4700-be79-26c5fdcbc68e.png)

**Comment by jdreesen** [comment]
Which Pimcore version are you using? The bundle is for Pimcore 5, current version is 10.

**Reply by akashhaldwani** [reply]
I'm using 10.5 version,
I was installed "User tracking (GDPR)" to check users' actions in the Pimcore administration panel.
and when I enabled the plugin than reported error coming.
